File: problems/3289-the-two-sneaky-numbers-of-digitville/solution.py
class Solution:
    def getSneakyNumbers(self, nums: List[int]) -> List[int]:
        count = {}
        for num in nums:
            count[num] = count.get(num, 0) + 1
        result = []
        for num, freq in count.items():
            if freq == 2:
                result.append(num)
        return result

File: problems/3289-the-two-sneaky-numbers-of-digitville/solution.java
class Solution {
    public int[] getSneakyNumbers(int[] nums) {
        Map<Integer, Integer> countMap = new HashMap<>();
        for (int num : nums) {
            countMap.put(num, countMap.getOrDefault(num, 0) + 1);
        }
        int[] result = new int[2];
        int index = 0;
        for (Map.Entry<Integer, Integer> entry : countMap.entrySet()) {
            if (entry.getValue() == 2) {
                result[index++] = entry.getKey();
            }
        }
        return result;
    }
}